src-tauri/src/mod_management/legacy.rs

```diff
@@ -115,6 +115,42 @@ pub fn parse_installed_mods(
     Ok(mods)
 }
 
+/// Deletes all legacy packages that match in author and mod name
```

Comment:
Could you be more precise regarding to what is a legacy package?

Reply:
Sure. "Legacy package" is what I call a Thunderstore package installed into `mods` the old-fashioned way, i.e. how we did it up to now as opposed to using the `packages` folder.

Should I update the comment with something like that? ^^

Reply:
Yep, please add this to the documentation :)

Reply:
Done with 448c6f343984b48b3ba7e1b9faaab54651b758f9